rcl/package.xml

```diff
@@ -24,6 +24,9 @@
   <depend>rosidl_runtime_c</depend>
   <depend>tracetools</depend>
 
+  <depend>rosidl_typesupport_introspection_c</depend>
```

Comment:
why do you need this dependency?
It doesn't seem that you're using this anywhere.

Reply:
There _was_ a dependency on `rosidl_typesupport_introspection_c` but that has since been removed as I iterated on this PR. I will remove the dependency.